python/gtsam/preamble.h

```diff
@@ -10,3 +10,6 @@ PYBIND11_MAKE_OPAQUE(std::vector<gtsam::Pose3>);
 PYBIND11_MAKE_OPAQUE(std::vector<boost::shared_ptr<gtsam::BetweenFactor<gtsam::Pose3> > >);
 PYBIND11_MAKE_OPAQUE(std::vector<boost::shared_ptr<gtsam::BetweenFactor<gtsam::Pose2> > >);
 PYBIND11_MAKE_OPAQUE(std::vector<gtsam::IndexPair>);
+PYBIND11_MAKE_OPAQUE(std::vector<gtsam::SfmMeasurement>);
```

Comment:
why do we need to add these here? Didn't catch what `PYBIND11_MAKE_OPAQUE` does for us

Reply:
This explains it well: https://pybind11.readthedocs.io/en/stable/advanced/cast/stl.html